scripts/settings-generator.js

```diff
@@ -78,6 +82,21 @@ Object.entries(providers).forEach(([name, desc], index) => {
 
   // Remove the properties from extended class.
   const providerKeys = Object.keys(schema.definitions[desc.type]['properties']);
+
+  // TODO: fix for ChromeAI
```

Comment:
Looks like https://github.com/jupyterlite/ai/pull/19 contains some changes to the `settings-generator` script that be useful here too.

Reply:
Looks like switching to `ChromeAIInputs` should fix it.